framework/components/ecl-page-headers/basic/README.md

```diff
@@ -0,0 +1,28 @@
+# Basic page header
+
+## Why and how to use this component
+Users need a way to easily identify the branding, to search and change the language if needed.
+the page title functions as the top level H1.
```

Comment:
Please either format as un-ordered list (bullets) as in the [docs](https://webgate.ec.europa.eu/CITnet/confluence/pages/viewpage.action?pageId=658281876) or start with capital letters on new sentences.
Currently it's like a single paragraph with sentences that does not start with capital letters:

...
Users need a way to easily identify the branding, to search and change the language if needed. the page title functions as the top level H1. it is also used to construct the
...